ouroboros-consensus-cardano-tools/app/immdb-server.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE ApplicativeDo   #-}
+{-# LANGUAGE NamedFieldPuns  #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Main (main) where
+
+import           Data.Void
+import qualified Network.Socket as Socket
+import           Options.Applicative
+
+import           Ouroboros.Consensus.Node.ProtocolInfo (ProtocolInfo (..))
+
+import           Cardano.Tools.DBAnalyser.Block.Cardano (Args (..))
+import           Cardano.Tools.DBAnalyser.HasAnalysis (mkProtocolInfo)
+import qualified Cardano.Tools.ImmDBServer.Diffusion as ImmDBServer
+
+main :: IO ()
+main = do
+  Opts {..} <- execParser optsParser
```

Comment:
Minor: don't we indent with 4 spaces regardless of whether there's a `where` clause?